KomodoPlatform/komodo-wallet-desktop
# Fix/dexpro/orderbook/bignum alignment

Closes https://github.com/KomodoPlatform/komodo-wallet-desktop/issues/2427

To test:
- login and activate MAZA
- go to pro view, and select KMD/MAZA for orderbook
- confirm no misalignment
